### PR TITLE
Sort pickup collectors by date

### PR DESCRIPTION
**Pull request by tiltec**
`collectors` doesn't honor the ordering, but `pickupdatecollector_set` solves it. 

I added a bit more to the API test that reverses the order, to test both ways and be more sure we don't rely on unspecified database order. I also removed the model test, as it doesn't add much over the API test (although it respects the [Test Pyramid](https://martinfowler.com/bliki/TestPyramid.html))

@lwm how does this look to you?

Closes https://github.com/yunity/karrot-frontend/issues/1157

### CODE REVIEW

File: foodsaving/pickups/models.py [hunk 1]
```diff
@@ -288,7 +288,7 @@ class PickupDateCollector(BaseModel):
     class Meta:
         db_table = 'pickups_pickupdate_collectors'
         unique_together = (('pickupdate', 'user'), )
-        ordering = ['-created_at']
+        ordering = ['created_at']
```
Comment (lwm) [inline foodsaving/pickups/models.py]: No migration needed?

Comment (tiltec) [reply foodsaving/pickups/models.py]: Ah right. It does create a migration, so I better commit it. But it doesn't have any effect on the database, so it works without.

File: foodsaving/conversations/api.py [hunk 2]
```diff
@@ -139,7 +139,7 @@ def list(self, request, *args, **kwargs):
         pickup_conversations = [item for item in conversations if item.target_type == pickup_ct]
         pickups = PickupDate.objects. \
             filter(id__in=[c.target_id for c in pickup_conversations]). \
-            prefetch_related('collectors', 'feedback_given_by')
+            prefetch_related('pickupdatecollector_set', 'feedback_given_by')
```
Comment (lwm) [inline foodsaving/conversations/api.py]: :clap: :clap: :clap:

**Comment by lwm** [comment]
LGTM :+1:

**Pull request merged**